.gitignore

```diff
@@ -6,3 +6,5 @@ contractLogFile.log
 .DS_Store
 lerna-debug.log
 test/.DS_Store
+package-lock.json
```

Comment:
as far as I remember the package-lock should not be in the ignore file, because it is requested by npm